packages/ui-library/src/components/radio-group/index.stories.ts

```diff
@@ -230,18 +228,17 @@ export const BlrRadioGroup = (params: BlrRadioGroupType) => BlrRadioGroupRenderF
 BlrRadioGroup.storyName = 'Radio Group';
 
 const defaultParams: BlrRadioGroupType & {
-  // direction: 'horizontal' | 'vertical';
   ariaLabel: string;
   radioGroupId: string;
 } = {
   theme: 'Light',
-  size: 'md',
-  // direction: 'horizontal',
-  hasGroupLabel: true,
-  groupLabel: 'Group Label Text',
+  sizeVariant: 'md',
+  direction: 'horizontal',
+  hasLegend: true,
+  legend: '"Legend-text"',
```

Comment:
Is double quote inside the single quote intentional?

Reply:
done